device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientOptions.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.device;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Options that allow configuration of the device client instance during initialization.
+ */
+public final class ClientOptions
+{
+    /**
+     * The Digital Twin Model Id associated with the device identity.
+     * Non plug and play users should not set this value
```

Comment:
FYI - we do not want to use the term "plug and play" in any public comments/ APIs.

Reply:
I am confused, are you suggesting to document this as Digital Twin users?

Reply:
I am not sure, `digital twin users` could potentially confuse users, since there is a different service called "digital twins". 
@prmathur-microsoft - any suggestions on which term to use here?

Reply:
exactly, that is the reason why I am asking.

Reply:
I would leave it at model id and revisit this when we have a better story around this.